app/src/main/AndroidManifest.xml

```diff
@@ -17,9 +17,10 @@
         android:icon="@mipmap/ic_launcher"
         android:label="@string/app_name"
         android:roundIcon="@mipmap/ic_launcher_round"
-        android:supportsRtl="true"
         android:screenOrientation="portrait"
+        android:supportsRtl="true"
```

Comment:
Please revert this, app don't has arabic/hebrew translations.

Reply:
Done